### PR TITLE
Add LTI 1.3 tool documentation

### PR DESCRIPTION
**Pull request by murhum1**
# Description

Documentation for setting up the A+ LTI 1.3 tool development environment.

# Testing

Created an environment from scratch based on these instructions and tested adding the tool in moodle, as well as adding an exercise and submitting one as a student to verify the environment works as expected.

### CODE REVIEW

File: doc/LTI_TOOL.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    environment:
+      POSTGRES_USER: moodle
+      POSTGRES_PASSWORD: "m@0dl3ing"
+      POSTGRES_DB: moodle
+    volumes:
+      - moodledb:/var/lib/postgresql/data
+```
+And the following line in `services.plus.environment`:
+```
+APLUS_LOCAL_SETTINGS: '/srv/aplus/a-plus/local_settings.py'
```
Comment (Mankro) [inline doc/LTI_TOOL.md:60]: Is this file path correct? There is no such file in the A-plus git repo. Does it mean that one should create this file oneself based on the new file `doc/lti_local_settings.py`?

**Review by Mankro — COMMENTED** [review]
Great! I have one question below before merging.

File: doc/LTI_TOOL.md (hunk 1 above)
Comment (murhum1) [reply doc/LTI_TOOL.md:60]: Yes, this is the intention - e.g. when mounting the user's own code, they should have the local_settings.py file there as well.

**Review by Mankro — APPROVED** [review]
Thanks!

**Pull request merged**